### ISSUE TITLE
Leagueprefs league website failing

### ISSUE THREAD

**Issue by Shteve0** [TheNAF/naflm#115]
Redirecting to /# instead of provided link

**Comment by rythos42** [comment]
Where is this link?

**Comment by Shteve0** [comment]
League Menu > League Forum. When I've installed it for local leagues,
updating the leagueprefs menu doesn't set this parameter properly - just
loops back to /#.

I've got tonight blocked out to take another look at the league preferences
menu, will dig into it then!
On 3/05/2016 6:15 am, "Craig Fleming" <notifications@github.com> wrote:

Where is this link?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub
<https://github.com/TheNAF/naflm/issues/115#issuecomment-216315864>
